docker/Dockerfile.cpu

```diff
@@ -4,7 +4,7 @@ LABEL Author="Ben Townsend, Madison May"
 # tensorboard
 EXPOSE 6006
 
-RUN apt-get update && mkdir /Finetune
+RUN apt-get update && apt-get install node.js && apt-get install npm && mkdir /Finetune
```

Comment:
these are needed to import huggingface's tokenizer library. Given the extra weight, I'd probably suggest not merging this PR unless we find xlm-roberta useful enough to warrant it.